tests/sharding2/data/db1.increment.sql

```diff
@@ -0,0 +1,6 @@
+create database sharding22;
+create table sharding22.t1 (id int primary key, name varchar(20));
+alter table sharding22.t1 add index abc(name);
+insert into sharding22.t1 values(1, "a"), (2, "b");
+insert into sharding22.t1 values(3, "c"), (4, "d");
+insert into sharding22.t1 values(5, "e"), (6, "f");
```

Comment:
How about adding more continuous ddls to test? IMO this bug is caused because of continuous ddls.
For example,
```sql
create table t3 (id bigint auto_increment,uid int,name varchar(20),primary key (`id`),unique key(`uid`)) DEFAULT CHARSET=utf8mb4;
alter table t3 add column c int;
alter table t3 add index c(c);
alter table t3 add column d int;
alter table t3 add index d(d);
alter table t3 add column e int, add index e(e);
```

Reply:
add in 30e8adb